src/js/media-preview-thumbnail.js

```diff
@@ -56,7 +63,8 @@ class MediaPreviewThumbnail extends window.HTMLElement {
     );
     if (mediaControllerId) {
       // @ts-ignore
-      this.#mediaController = this.getRootNode()?.getElementById(mediaControllerId);
+      this.#mediaController =
+        this.getRootNode()?.getElementById(mediaControllerId);
```

Comment:
nitpick(blocking): whoops lost your @ts-ignore by the formatting! either move back to one line or move `//@ts-ignore`